salt/orchestrate/services/elasticsearch.sls

```diff
@@ -69,12 +72,12 @@ deploy_elasticsearch_nodes:
 sync_external_modules_for_elasticsearch_nodes:
   salt.function:
     - name: saltutil.sync_all
-    - tgt: 'G@roles:elasticsearch and G@environment:{{ ENVIRONMENT }} and G@launch-date:{{ launch_date }}'
+    - tgt: '{{ target_string }} and G@launch-date:{{ launch_date }}'
```

Comment:
Why are we keeping the `launch-date` here unlike the rest of them?

Reply:
That's because with elasticsearch and Rabbit we might be running the state to expand an existing cluster, in which case we only want to highstate the newly launched instances.